pkg/ctl/install/fluxinstaller.go

```diff
@@ -0,0 +1,558 @@
+package install
```

Comment:
This could go into its own package outside of `pkg/ctl`

Reply:
Per earlier discussion I've opened #1076.

Reply:
Then we can solve it when #1076 is tackled, or should I address it now?

Reply:
Perhaps, just looks like it is a new file and it is not that much effort to structure code from the start. Clearly something is wrong with the diff...

Reply:
What do you mean? I don't think there is anything wrong with the diff.